src/cluster/manager_common_updater.go

```diff
@@ -282,7 +282,6 @@ func (c *commonUpdater) Update(ctx context.Context) error {
 }
 
 // labelNodesWithNodePoolName add node pool name labels for all nodes.
-// It's used only used in case of ACK etc. when we're not able to add labels via API.
 func labelNodesWithNodePoolName(ctx context.Context, commonCluster CommonCluster) error {
```

Comment:
do we use this anywhere?

Reply:
Just above in line 278 inside the Update func.